src/cljs/athens/views/buttons.cljs

```diff
@@ -55,7 +55,7 @@
                              :text-align "left"}]
                      [:kbd {:margin-inline-start "1rem"
                             :font-size "85%"}]
-                     [:svg button-icons-style
+                     ["> svg" button-icons-style
```

Comment:
What's the difference here?

Reply:
`>` allows us to target only an SVG that's a direct descendant of the current context, and not any deeper ones.